### ISSUE TITLE
'Functional' object has no attribute '_layers'??

### ISSUE THREAD

**Issue by Akorex** [keras-team/keras-core#756]
While trying to build a model using a pre-trained vgg16 model as a base model for feature extraction, i ran into this error `'Functional' object has no attribute '_layers'` while trying to compile the model.

Code which led to error:

`
model = models.Sequential()

model.add(conv_base)

model.add(layers.Flatten())

model.add(layers.Dense(256, activation='relu'))

model.add(layers.Dropout(0.5))

model.add(layers.Dense(512, activation='relu'))

model.add(layers.Dropout(0.5))

model.add(layers.Dense(len(class_names), activation='softmax'))
`

I was able to run model.summary() with the model created but error occured at model.compile()...

This occured on the JAX & TensorFlow backends.

**Comment by fchollet** [comment]
Thanks for the report -- please provide a standalone, reproducible code snippet, such as a Colab notebook.

**Comment by Akorex** [comment]
Colab Notebook: https://colab.research.google.com/drive/15EjuAYXYlKxh3YJaNJ4ud8LFYFKsKspf?usp=sharing

**Comment by fchollet** [comment]
The issue is that you are using a `tf.keras` model instead of a Keras Core model.

Replace the line

```python
from tensorflow.keras.applications import VGG16
```

with

```python
from keras_core.applications import VGG16
```
